src/frontend/core/executor/impl/StreamingTriangleCountExecutor.cpp

```diff
@@ -369,12 +370,19 @@ long StreamingTriangleCountExecutor::aggregateCentralStoreTriangles(
 
     std::vector<std::string> triangles = Utils::split(result, ':');
     std::vector<std::string>::iterator triangleIterator;
+    std::set<std::string> uniqueTriangleSet;
+
     long currentSize = triangleCount;
 
     for (triangleIterator = triangles.begin(); triangleIterator != triangles.end(); ++triangleIterator) {
         std::string triangle = *triangleIterator;
 
         if (!triangle.empty() && triangle != "NILL") {
+            if (runMode == "0") {
```

Comment:
What is this `runMode` mean?

Reply:
runMode defines the method of streaming triangle counting. If runMode is 0, then triangle counting is done from scratch. If it's 1, then triangle counting is done incrementally, only for new edges.